packs/data/equipment.db/dagger-of-venom.json

```diff
@@ -20,7 +20,7 @@
             "value": ""
         },
         "description": {
-            "value": "<p>The serrated blade of this <em>+1 striking</em> dagger has a greenish tinge, and the hilt is sculpted to look like the head of a serpent about to strike.</p>\n<p>When you critically succeed at an attack roll with the dagger of venom, the target becomes sickened 1 unless it succeeds at a DC 19 Fortitude save. This is a poison effect.</p>\n<p>In addition, you can activate the dagger to poison a creature with a more potent poison.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">F</span> Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You damage a creature with the <em>dagger of venom</em></p>\n<hr />\n<p><strong>Effect</strong> You poison the creature you hit with dagger venom.</p>\n<p><strong>Dagger Venom</strong> (poison)</p>\n<p><strong>Saving Throw</strong> Fortitude DC 21</p>\n<p><strong>Maximum Duration</strong> 4 rounds.</p>\n<p><strong>Stage 1</strong> 1d8 poison damage and enfeebled 1.</p>"
+            "value": "<p>The serrated blade of this <em>+1 striking</em> dagger has a greenish tinge, and the hilt is sculpted to look like the head of a serpent about to strike.</p>\n<p>When you critically succeed at an attack roll with the dagger of venom, the target becomes @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} unless it succeeds at a @Check[type:fortitude|dc:19|traits:poison] save. This is a poison effect.</p>\n<p>In addition, you can activate the dagger to poison a creature with a more potent poison.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">F</span> Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You damage a creature with the <em>dagger of venom</em></p>\n<hr />\n<p><strong>Effect</strong> You poison the creature you hit with dagger venom.</p>\n<p><strong>Dagger Venom</strong> (poison)</p>\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:21]</p>\n<p><strong>Maximum Duration</strong> 4 rounds.</p>\n<p><strong>Stage 1</strong> [[/r {1d8}[poison]]]{1d8 poison damage} and @Compendium[pf2e.conditionitems.Enfeebled]{Enfeebled 1}.</p>"
```

Comment:
```suggestion
            "value": "<p>The serrated blade of this <em>+1 striking</em> dagger has a greenish tinge, and the hilt is sculpted to look like the head of a serpent about to strike.</p>\n<p>When you critically succeed at an attack roll with the dagger of venom, the target becomes @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} unless it succeeds at a @Check[type:fortitude|dc:19|traits:poison] save. This is a poison effect.</p>\n<p>In addition, you can activate the dagger to poison a creature with a more potent poison.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">F</span> Interact</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You damage a creature with the <em>dagger of venom</em></p>\n<hr />\n<p><strong>Effect</strong> You poison the creature you hit with dagger venom.</p>\n<p><strong>Dagger Venom</strong> (poison)</p>\n<p><strong>Saving Throw</strong> @Check[type:fortitude|dc:21|traits:poison]</p>\n<p><strong>Maximum Duration</strong> 4 rounds.</p>\n<p><strong>Stage 1</strong> [[/r {1d8}[poison]]]{1d8 poison damage} and @Compendium[pf2e.conditionitems.Enfeebled]{Enfeebled 1}.</p>"
```
Added poison trait to the second inline save

Reply:
The dagger has the poison trait on the item no?